README.md

```diff
@@ -1,83 +1,130 @@
+It's completely possible to install with
+[straight.el](https://github.com/raxod502/straight.el),
+[use-package](https://github.com/jwiegley/use-package),
+[doom](https://github.com/hlissner/doom-emacs), etc. Instructions for doing so
+are beyond the scope of this README, but note that `autoloads` are implemented
+for entry-point functions so if you like to have a speedy start-up time you can
+use the `:defer t` option.
 
-Put all `*.el` files in emacs' load-path. e.g. if you put them in
-`~/emacs/`, add the following lines to `~/.emacs` (or whatever your init
-file is called):
+### Install Option 2: Debian package
 
-```
-(add-to-list 'load-path "~/emacs")
-(require 'sclang)
-```
+There is a debian package which provides emacs integration called
+`supercollider-emacs`. Option #1 will likely be more recent, but 
+if you prefer you can install the package with:
 
-for the HTML help system to fully function also add
-```
-(require 'w3m)
+``` shell
+sudo apt install supercollider-emacs
 ```
 
-now put all `*.sc` files in sclang's library path, e.g. if you put them
-in a non-standard location, such as `~/SuperCollider/Emacs`, add the
-following to `~/.config/SuperCollider/sclang_conf.yaml` (Linux) or `~/Library/Application Support/SuperCollider/sclang_conf.yaml` (macOS):
+### Install Option 3: Installing from source
 
-```
-includePaths:
-    [~/SuperCollider/Emacs]
-```
+If you are building SuperCollider from source, you can optionally compile and
+install this library along with it. The cmake `-DSC_EL` flag controls whether
+scel will be compiled. On Linux machines `-DSC_EL=ON` by default. See the
+supercollider README files for more info.
 
-(note normally this is not needed as they are put into sclang's library
-path during installation with scons).
+## Installation requirements
 
+For the HTML help system, you will need emacs-w3m support, but you can still use
+the rest of the functionality without the w3m dependency.
 
-Usage
------
+```emacs-lisp
+;; in ~/.emacs
+(require 'w3m)
+```
 
-In order to automatically start sclang when invoking emacs, use the following command line:
+## Usage
 
-```
-$> emacs -sclang
+The main function which starts interacting with the sclang interpreter is
+`sclang-start`. You can execute that anywhere with `M-x sclang-start`, or from
+within a `.scd` buffer by pressing `C-c C-o`.
+
+If you know you want to launch sclang when you start emacs you can use the `-f`
+option to execute that function right away:
+
+``` shell
+# in your terminal
+emacs -f sclang-start
 ```
 
-you're now ready to edit, inspect and execute sclang code!
+## Configuration
+
+To fine-tune the installation from within emacs' graphical customization
+interface, type:
+
+`M-x sclang-customize`
 
+If you want to change from the default `sclang_conf.yaml` file used by your
+emacs sessions you can customize the `sclang-library-configuration-file`
+variable, but make sure that file loads the scel quark, or you won't be
+able to run sclang code.
```

Comment:
I ended up using your text.

I was able to set the value by selecting `1 file` from the menu and then tabbing over and filling in the file field with the filename. Screenshot below

<img width="624" alt="Screen Shot 2021-09-04 at 9 16 59 AM" src="https://user-images.githubusercontent.com/11772/132097214-4aaf9320-a0a6-4ec8-b04a-f2c806fff1bb.png">

In my opinion this experience is a bit clunky and I think it can be improved by changing the `defcustom` options. If it's ok with you I'd prefer to do that in a separate PR.

Reply:
Perfect, thanks so much!

> I was able to set the value by selecting 1 file from the menu and then tabbing over and filling in the file field with the filename. Screenshot below

That sounds great, I thought this might be something on my end so if you were able to set this, we're all set.

> In my opinion this experience is a bit clunky and I think it can be improved by changing the `defcustom` options. If it's ok with you I'd prefer to do that in a separate PR.

I agree on both accounts - clunky experience, for another PR.